### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'response')

### ISSUE THREAD

**Issue by cryptic-widow** [Iku/Google-Forms-to-Discord#134]
This script has worked beautifully up until yesterday. Absolutely no changes have been made and I'm not sure what has happened here.

I went into the script editor and clicked "run" to see what the execution log would say and I got this error:
> 7:12:20 AM	Error	
> TypeError: Cannot read properties of undefined (reading 'response')
> onSubmit	@ Code.gs:4

Debug shows this:
![Capture](https://user-images.githubusercontent.com/83616610/219374928-ab37168a-28c8-49ba-ae13-3979212f56e9.PNG)
(redacted webhook url)

**Comment by Kelo** [comment]
It's mostlikely because you clicked run instead of just submitting a response.
The "Run" button will try to take the latest response from the form, but if the form hasn't received a response (ever or in a while) it won't have a response to run your code through.

TLDR; Try submitting a response instead of hitting the "Run" button
